amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ConnectionContainer.java

```diff
@@ -30,19 +30,17 @@
 @Slf4j
 public class ConnectionContainer {
 
-    private ExchangeContainer exchangeContainer;
-    private static Map<NamespaceName, Set<AmqpConnection>> connectionMap = Maps.newConcurrentMap();
+    private Map<NamespaceName, Set<AmqpConnection>> connectionMap = Maps.newConcurrentMap();
 
-    public ConnectionContainer(PulsarService pulsarService, ExchangeContainer exchangeContainer) {
-        this.exchangeContainer = exchangeContainer;
+    public ConnectionContainer(PulsarService pulsarService,
```

Comment:
Should this constructor be public?  Is it supposed to be created from outside this package?  Or, only through the `AmqpBrokerService`?

Reply:
protected is better :)